src/main/java/mate/academy/spring/controller/UserController.java

```diff
@@ -22,7 +23,7 @@ public UserController(UserService userService,
     }
 
     @GetMapping("/by-email")
-    public UserResponseDto findByEmail(@RequestParam String email) {
+    public UserResponseDto findByEmail(@RequestParam @Valid String email) {
```

Comment:
does it work? do we need this annotation here?
maybe we can use @Email annotation instead?

https://medium.com/codex/spring-boot-create-custom-annotation-to-validate-request-parameter-dcf483539d90#:~:text=Annotate%20the%20Controller%20with%20%40Validated%20annotation%2C%20otherwise%20the%20annotation%20ValuesAllowed%20will%20not%20be%20triggered%20at%20all.

let's remove it or implement properly (see link)